packages/components/src/components/hds/modal/index.ts

```diff
@@ -69,7 +69,7 @@ export default class HdsModal extends Component<HdsModalSignature> {
     super(owner, args);
 
     registerDestructor(this, (): void => {
-      document.removeEventListener('click', this._clickHandler);
+      document.removeEventListener('click', this._clickHandler, true);
```

Comment:
No way! 🤯

Reply:
[question]: since the listener is added as:
```
document.addEventListener('click', this._clickHandler, {
  capture: true,
  passive: false,
});
```
should also be removed in the same way (just in case)?
```
document.removeEventListener('click', this._clickHandler, {
  capture: true,
  passive: false,
});
```

Reply:
Ok this is a good question, looking at mdn: they only list capture as an available option ([docs](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener#options)), but they have an example that sets passive: false (https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener#matching_event_listeners_for_removal).

But ultimately they recommend matching the values when you add / remove the listener so I'll update.